pkg/sync/v7/translate.go

```diff
@@ -310,48 +304,6 @@ var translations = map[string][]struct {
 			Template: "{{ .Config.Images.ServiceCatalog }}",
 		},
 	},
-	"DaemonSet.apps/openshift-azure-logging/mdsd": {
```

Comment:
How sure are you that an upgrade from v6 to this PR has a working logging stack at the end of it?

Reply:
I'm going to make sure this passes the upgrade-v6.0 tests (which are mysteriously missing from CI).

We know that we have a working logging stack on a 5.1/5.2 -> HEAD upgrade so I'm relatively confident...